.github/workflows/build_linux_packages.yml

```diff
@@ -110,13 +115,15 @@ jobs:
           key: linux-build-packages-manylinux-v2-${{ github.sha }}
 
       - name: Configure AWS Credentials
+        if: always()
         uses: aws-actions/configure-aws-credentials@ececac1a45f3b08a01d2dd070d28d111c5fe6722 # v4.1.0
         with:
           aws-region: us-east-2
           role-to-assume: arn:aws:iam::692859939525:role/therock-artifacts
 
       # TODO: Move to script
       - name: Create Index Files
+        if: always()
```

Comment:
As you're assuming that the `logs` directory might not exist, it might be worth to rather guard this case here as the line `python build/indexer.py -f '*.log' build/logs/` as well as the call to sed wont work then.

Reply:
Sounds good - handled that